src/components/Header.js

```diff
@@ -15,7 +16,7 @@ const Header = () => (
         </ul>
       </div>
       <div className="user-icon">
-        <p>User icon</p>
+        <img src={userIcon} alt="user icon" />
       </div>
     </nav>
   </>
```

Comment:
- Hey, Please, let's improve our design to make it more close to the original design. see the images below to improve your deign.
   - Kindly aligh the percentage counter correctly.
   - Add the correct space behind the Bookstore CMS
   - Add the correct space at the end of the icon

*main design*
![image](https://github.com/NoelLincoln/bookstore/assets/59397159/0c739731-6cf4-4196-8831-811883a52c39)


*your design*
![image](https://github.com/NoelLincoln/bookstore/assets/59397159/2338308d-50f4-4824-b73c-d24ee8520d09)